development/docker-compose-deps.yml

```diff
@@ -40,6 +41,8 @@ services:
     labels:
       namespace: "prefect"
       pod: "prefect-server"
+    volumes:
+      - ../:/source
```

Comment:
I think this should be in `development/docker-compose.local-build.yml`

Reply:
Hm, yes that seems reasonable. Though it seems like `development/docker-compose.local-build.yml` is only optionally included with `build_compose_files_cmd()` and not `build_dev_compose_files_cmd()`. I'll add another `development/docker-compose.local-build-deps.yml` file for that.

Reply:
I've pushed an update to move this volume to `docker-compose.local-build-deps.yml` instead.